src/CodeGen_X86.cpp

```diff
@@ -480,14 +486,16 @@ void CodeGen_X86::codegen_vector_reduce(const VectorReduce *op, const Expr &init
         Expr pattern;
         const char *intrin;
         Type narrow_type;
+        bool combine_init;
```

Comment:
This may be better as a set of enum flags, similar to [CodeGen_PTX_Dev](https://github.com/halide/Halide/blob/32d5f71cef5ecea883c6fc26305181e555716d01/src/CodeGen_PTX_Dev.cpp#L444)